app/logic/Mail/ActiveSync/ActiveSyncFolder.ts

```diff
@@ -31,6 +31,7 @@ export enum FolderType {
 export class ActiveSyncFolder extends Folder implements ActiveSyncPingable {
   account: ActiveSyncAccount;
   messages: EMailCollection<ActiveSyncEMail>;
+  readonly newMessageCallbacks = new ArrayColl<{(message: ActiveSyncEMail): boolean}>();
```

Comment:
```suggestion
  /** Callbacks that are called when new messages arrive in this folder.
   * If the callback returns true, it will be removed and not be called anymore. */
  readonly newMessageCallbacks = new ArrayColl<{(message: ActiveSyncEMail): boolean}>();
```